server/service/src/invoice/prescription/update/mod.rs

```diff
@@ -321,16 +322,15 @@ mod test {
         // Test all fields apart from status
         fn get_update() -> UpdatePrescription {
             UpdatePrescription {
-                id: prescription().id,
-                status: None,
-                patient_id: Some(mock_patient_b().id),
-                clinician_id: Some(NullableUpdate {
-                    value: Some(clinician().id),
-                }),
-                comment: Some("test_comment".to_string()),
-                colour: Some("test_colour".to_string()),
-                backdated_datetime: None,
-                diagnosis_id: None,
+                id:prescription().id,
```

Comment:
Looks like formatting got a bit messed up here. Can you run the formatter on this file?